faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
```

Comment:
I don't actually see anything in this FAQ about feature requests? So maybe we just make the title "Submitting a bug report"

Please note that submitting a bug report is already part of the [Troubleshooting FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html), so please think about how best to align those two and avoid duplication.

Reply:
Yes, I have viewed the content of the FAQ that you have mentioned. I have ensured to reduce the redundancy of content.

Reply:
> Please note that submitting a bug report is already part of the [Troubleshooting FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html), so please think about how best to align these two and avoid duplication.

Ah yes! I did not check! And when I search https://training.galaxyproject.org/training-material/search?query=Troubleshooting it does not find it. Better to look at https://training.galaxyproject.org/training-material/faqs/galaxy/! Thanks.

Reply:
@annefou, oh, good to know it's now showing up in the search, will look into why that is :/